casanovo/denovo/dataloaders.py

```
@@ -342,6 +355,288 @@ def _make_dataset(
+                        : self.scan_width
+                    ]
+                    ms1_rts = ms1_rts[sorted_ms1_rt_idxs]
+                    ms1_scans = ms1_scans[sorted_ms1_rt_idxs]
+
+                    for charge in range(1, self.max_charge + 1):
+                        mz_array = []
+                        intensity_array = []
+                        scan_window_array = []
+                        ms_array = []
+
+                        for scan, cur_rt in zip(scans, rts):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)